core/domain/exp_jobs_one_off.py

```diff
@@ -831,3 +831,63 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class EmptySubtitledHtmlAuditJob(jobs.BaseMapReduceOneOffJobManager):
+    """One off job to audit explorations having empty subtitled html and outputs
+    the content_id of such object.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        exp_rights = rights_manager.get_exploration_rights(item.id)
+
+        state_name_to_content_id_list = {}
+        for state_name, state in exploration.states.iteritems():
+            empty_html_content_id_list = []
+            content = state.content
+            if not (content.html).strip():
```

Comment:
Don't need the parens around `content.html` here.

Reply:
Done!